airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -9,6 +9,7 @@ import Status from "core/statuses";
 
 import { useAttemptLink } from "./attemptLinkUtils";
 import ContentWrapper from "./components/ContentWrapper";
+import ErrorDetails from "./components/FailureDetails";
```

Comment:
nit: if we're importing this as `ErrorDetails` and exporting it as such within the file, should the file be named `ErrorDetails`?